test/apps/iodemo/io_demo.cc

```diff
@@ -678,11 +683,14 @@ class DemoClient : public P2pDemoCommon {
                                  0, opts().operations.size() - 1)];
     }
 
-    inline void check_time_limit(double current_time) {
+    inline bool check_time_limit(double current_time) {
```

Comment:
maybe no need to return bool: update_retry can check `if (_status != OK)` return true

Reply:
yes, makes sense
done